### PR TITLE
Reduce the noise when running unit tests, fixing warnings

### LINKED ISSUE #485: Fix vue-test-utils deprecation warnings before they release a version that removes these functions
**Describe the bug**

Spent most of the day running unit and e2e tests, and noticed several things that could be improved. Amongst these things, there are some vue-test-utils warnings. If we have time, it's better to fix these as otherwise after updating dependencies one would have to chase for errors that may not be as clear as these messages (i.e. not easy to relate the error to the message).

```
[vue-test-utils]: contains is deprecated and will be removed in the next major version. Use `wrapper.find`, `wrapper.findComponent` or `wrapper.get` instead.
[vue-test-utils]: finding components with `findAll` is deprecated and will be removed in the next major version. Use `findAllComponents` instead.
[vue-test-utils]: finding components with `findAll` is deprecated and will be removed in the next major version. Use `findAllComponents` instead.

```

**Release version(s) and/or repository branch(es) affected?**

**Steps to reproduce the bug**

Run unit tests.

**Expected behavior**

No warnings.

**Screenshots**

**Additional context**

**Pull requests welcome!**
This is an Open Source project - please consider contributing a bug fix
yourself (please read `CONTRIBUTING.md` before starting any work though).

### PR DESCRIPTION
**Pull request by kinow**
These changes close #485 #486 

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Already covered by existing tests.
- [x] No change log entry required (why? e.g. invisible to users).
- [x] No documentation update required.

### CODE REVIEW

**Comment by kinow** [comment]
Ready for review after #483 , as the remaining warnings for Tree and TreeItem components are fixed in that PR.

File: package.json [hunk 1]
```diff
@@ -93,6 +94,7 @@
     "standard": "^14.3.4",
     "subscriptions-transport-ws": "^0.9.18",
     "svgo": "^1.3.2",
+    "utf-8-validate": "^5.0.2",
```
Comment (kinow) [inline package.json:97]: I think one review will do. The only files changed in this PR are the test files, and these dependencies in `package.json`. These were added under the `devDependencies`, not used in the final artefact.

Just to suppress warnings that appeared when you ran `yarn run test:unit` :+1:

**Comment by kinow** [comment]
Can be reviewed with GitHub UI. Just look at an old run of actions, like this one from filter tree: https://github.com/cylc/cylc-ui/runs/1048768963?check_suite_focus=true (expand "Test").

image

Then compare with the actions console output for this PR.

image

**Review by oliver-sanders — APPROVED** [review]
No warnings locally or on GHA.

**Pull request merged**